tests/integration/springer/test_dag_pull_ftp.py

```diff
@@ -52,50 +52,134 @@ def test_dag_run(dag: DAG):
     assert repo.find_all() == expected_files
 
 
-def test_dag_migrate_from_FTP():
+def test_dag_trigger_file_processing():
+    repo = SpringerRepository()
+    assert list(map(lambda x: x["xml"], repo.find_all())) == trigger_file_processing(
+        "springer", repo, get_logger().bind(class_name="test_logger")
+    )
+
+
+def __test_dagrun_state(dagrun: DagRun):
```

Comment:
why you need it as a private function?

Reply:
because it's used inside the test:
https://github.com/cern-sis/workflows/blob/5c8b56a04083118b8a4ecd414ea5192e6d0e00bb/tests/integration/springer/test_dag_pull_ftp.py#L37